tests/core/memory/test_memory.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+from evm.vm.memory import (
+    Memory,
+)
+from evm.exceptions import (
+    ValidationError,
+)
+
+
+@pytest.fixture
+def memory():
+    return Memory()
+
+
+@pytest.fixture
+def memory32():
+    memory = Memory()
+    memory.extend(0, 32)
+    return memory
+
+
+@pytest.mark.parametrize("start_position,size,value", ((0, 4, b'1010'),))
```

Comment:
Not quite.  Parametrization with a single value strikes me as awkward since the idea behind parametrization is running the same test multiple times with different parameters.  In this case, if there is  only a single parameter, I think it's best just to have those values be part of the test.